lib/transformer.dart

```diff
@@ -474,12 +475,12 @@ class InitializerData {
 }
 
 // Reads a file list from a barback settings configuration field.
-_readFileList(BarbackSettings settings, String field) {
+List<String> _readFileList(BarbackSettings settings, String field) {
   var value = settings.configuration[field];
   if (value == null) return null;
-  var files = [];
+  List<String> files = [];
   bool error;
-  if (value is List) {
+  if (value is List<String>) {
```

Comment:
I've left the type casting instead of the inference, let me know if you prefer the other way.